tests/functional/test_scoping/test_global_import_modules.py

```diff
@@ -151,3 +153,41 @@ def test_globalimports_basic_test_with_distributed_model():
     #################################
     # END
     #################################
+
+
+def test_globalimports_with_project_root_model_parameter():
+    """
+    Basic test for the FQNGlobalRepo.
+    Tests that "project_root" model parameter has an effect.
+    """
+    #################################
+    # META MODEL DEF
+    #################################
+
+    my_meta_model = metamodel_from_file(
+        join(abspath(dirname(__file__)), 'interface_model2',
+             'Interface.tx'))
+    my_meta_model.register_scope_providers(
+        {"*.*": scoping_providers.FQNGlobalRepo(
+            join("model_a", "*.if"))})
+    my_meta_model._tx_model_param_definitions.add(
+        "project_root", "the project root")
```

Comment:
`project root` is a built-in param so language designer shouldn't need to define it explicitely.

Reply:
ok